### PR TITLE
Bug 1182297 - Platform selector does not have padding 

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

File: src/media/css/elements--select.styl [hunk 1]
```diff
@@ -31,6 +31,10 @@ mkt-selected {
     display: flex;
     height: 60px;
     justify-content: center;
+
+    p {
+        padding: 0 10px;
```
Comment (mstriemer) [inline src/media/css/elements--select.styl]: Can you change this to `padding: 0 15px;` so it lines up with the text will line up with the top-left menu button.

File: src/media/css/elements--select.styl [hunk 2]
```diff
@@ -42,10 +46,12 @@ mkt-selected:after {
     transition: transform .2s linear;
 
     +ltr() {
-        margin-left: 6px;
+        margin-left: 16px;
```
Comment (mstriemer) [inline src/media/css/elements--select.styl]: You can remove these `margin-left` and `margin-right` lines after bumping up the `padding` above.

**Comment by mstriemer** [comment]
I see an extra unrelated commit in these changes. Can you remove that from the branch too, please.

**Comment by tsl143** [comment]
Created new PR for this 
https://github.com/mozilla/fireplace/pull/1387

**Pull request closed without merging**